alphazero/_boardconverters.py

```diff
@@ -0,0 +1,19 @@
+import torch
+
+
+class BoardConverter:
+    @staticmethod
+    def board_to_tensor(board):
+        raise NotImplementedError
```

Comment:
can you provide a docstring explaining what this should do?

Reply:
Yup